### PR TITLE
Update Cirros to 1.3.3

### PR DESCRIPTION
**Pull request by ewindisch**

### CODE REVIEW

File: library/cirros [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # maintainer: Eric Windisch <ewindisch@docker.com> (@ewindisch)
 
-latest: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
+latest: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
+0.3.3: git://github.com/ewindisch/docker-cirros@5ef3f5024b0aa80553cc34be9eff6685cb31b458
 0.3.0: git://github.com/ewindisch/docker-cirros@1cded459668e8b9dbf4ef976c94c05add9bbd8e9
```
Comment (yosifkit) [inline library/cirros]: Is there a reason for keeping the old version, 0.3.0, around?  Is it going to be a [supported version](https://registry.hub.docker.com/_/cirros/)?

Comment (ewindisch) [inline library/cirros]: It seems reasonable to keep the N-1 version around, if users might depending on it run into issues/bugs.

Comment (tianon) [inline library/cirros]: We don't delete removed tags -- they just don't won't be rebuilt and aren't
officially supported.

Comment (tianon) [inline library/cirros]: See https://registry.hub.docker.com/_/redis/ vs
https://registry.hub.docker.com/_/redis/tags/manage/ for a good
example of this in practice.

Comment (ewindisch) [inline library/cirros]: That's fine & reasonable. I'll remove 0.3.0 then.

I had seen other official images keeping clearly-not-supported versions tagged here, so this wasn't clear. I'll follow-up on that and see about having those images remove their unsupported versions, too.

Comment (tianon) [inline library/cirros]: Indeed -- if you have specific examples, that would be helpful.  Some of
them have legitimate reasons for it that we've discussed with them.

**Comment by tianon** [comment]
LGTM

Build test of #450; c0633f08921f4e8f945190da1836999012d9e9b1 (`cirros`):

```console
$ bashbrew build https://raw.githubusercontent.com/docker-library/official-images/c0633f08921f4e8f945190da1836999012d9e9b1/library/cirros
Fetching cirros (git://github.com/ewindisch/docker-cirros) ...
Processing cirros:latest ...
Processing cirros:0.3.3 ...
```

**Comment by tianon** [comment]
Do you happen to have any insight into #390? (ie, why does CirrOS use `MST`?)

**Comment by ewindisch** [comment]
@tianon I've updated #390 directly. Thanks for the heads up

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**